cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -58,7 +58,7 @@ function Install-Binary($install_args) {
   Initialize-Environment
```

Comment:
ErrorActionPreference no longer should be overriden above and below in this function